department-of-veterans-affairs/va.gov-cms
# VACMS-7074 KB theme in vagovclaro

## Description

Relates to #7074. Some light theming around the search form in the header, based on this figma: https://www.figma.com/file/PySxm5aQeWCQvYqZpdkYJj/KB-content-design?node-id=257%3A133

KB still needs some love post-claro turn on, but this should get relative parity with what the current theme is providing with some slight improvements. CMS Sprint planning determined KB is probably the lowest priority thing for Claro turn on at this time, so won't be revisiting this in vagovclaro for a few weeks probably.

## Testing done
Viewed `/help/` `/help/search` and `/help/*/*` (various sub pages) to see that sidebar menu + header search form rendered correctly in all cases. Does not affect the search form on the `/help/` page, this is currently controlled by its own layout builder layout. The `/help/search` page is a view, that's its own rabbit hole.

## Screenshots
<img width="1345" alt="Screen Shot 2022-01-03 at 4 38 04 PM" src="https://user-images.githubusercontent.com/11279744/147995765-eff2c076-057a-474d-aa7a-96000d7b3386.png">

## QA steps

As user,
- visit `/help/` and see the search form printed in the page by default.
- visit and sub KB page and see the search form printed in the header area with some extra styling applied.
- Use the search form ("log in" has been my test query of choice) and see that the help/search view still populates results as expected, the form isn't receiving extra styling on this page.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?

vagovclaro only, no announcement needed
